pkg/formatted/color.go

```diff
@@ -20,15 +20,55 @@ import (
 	"github.com/fatih/color"
 )
 
+var (
+	allColors = []color.Attribute{
+		color.FgHiGreen,
+		color.FgHiYellow,
+		color.FgHiBlue,
+		color.FgHiMagenta,
+		color.FgHiCyan,
+		color.FgHiWhite,
+	}
+)
+
+type RainbowColors struct {
+	current int
+	cached  map[string]color.Attribute
+	all     []color.Attribute
+}
+
+func NewRainbowColors() *RainbowColors {
+	return &RainbowColors{
+		current: 0,
+		cached:  make(map[string]color.Attribute),
+		all:     allColors,
+	}
+}
+
+func (rc *RainbowColors) Get(label string) *color.Color {
+	if _, ok := rc.cached[label]; !ok {
+		rc.cached[label] = rc.all[rc.current]
+		rc.current++
```

Comment:
There might be a race here if this is called in different goroutine 🤔

Reply:
yeah i tough about that but since we werent using it i didn't follow tough, what do you suggest? external lib for locking etc...

Reply:
and fyi i did test it with `-race` and it was running well but i'll just sync/lock/release it in case off,

Reply:
@chmouel right, we might not be in the case of race condition in the cli, but if somebody uses this might hit it